src/main/resources/io/jenkins/plugins/orka/OrkaAgent/configure-entries.jelly

```diff
@@ -80,6 +80,29 @@
     <f:entry title="${%Name Prefix}" field="namePrefix" help="${descriptor.getHelpFile('namePrefix')}">
       <f:textbox/>
     </f:entry>
+
+    <orka:blockWrapper>
+      <f:entry title="Port Mapping" field="portMappings" description="Please enter the NODE and VM ports for port mapping. Only NODE ports between 1024 and 65535 are allowed.">
+        <f:repeatable var="entry" name="portMappings">
+            <f:entry title="Node Port" field="from">
```

Comment:
Should we add validation for the port? Right now I can set it to 2 and no warning/error is shown:

![Uploading image.png…]()